README.md

```diff
@@ -278,20 +271,29 @@ Ruby types:
 ~~~
 
 Rice includes default specializations for many of the builtin
-types. To define your own conversion, write a template specialization:
+types. To define your own conversion, you need to create two 
+class template specializations:
 
 ~~~{.cpp}
-  template<>
-  Foo from_ruby<Foo>(Object x)
+  namespace Rice::detail
   {
-    // ...
-  }
-
-  template<>
-  Object to_ruby<Foo>(Foo const & x)
-  {
-    // ...
-  }
+    template<>
+    structure From_Ruby<Foo>
```

Comment:
Did `structure` get added as a keyword or is this supposed to be `struct` that got autocompleted?

Reply:
Good catch - should be struct. Will fix that.